### PR TITLE
DEVX-2804 - Remove CLI tutorial 

### PR DESCRIPTION
**Pull request by joel-hamill**
### Description 

This tutorial is being moved to [docs-confluent-cli](https://github.com/confluentinc/docs-confluent-cli/pull/613).

### CODE REVIEW

File: ccloud/docs/beginner-cloud.rst [hunk 1]
```diff
@@ -2,35 +2,18 @@
...
+credentials, and ACLs.
 
 Prerequisites
 -------------
 
--  `Confluent Cloud <https://www.confluent.io/confluent-cloud/tryfree/>`__
+-  `Confluent Cloud <https://www.confluent.io/get-started/?product=cloud>`__
 
 -  Local `install of Confluent CLI
    <https://docs.confluent.io/confluent-cli/current/install.html>`__ (v3.0.0 or later)
```
Comment (tikimims) [inline ccloud/docs/beginner-cloud.rst]:
```suggestion
   <https://docs.confluent.io/confluent-cli/current/install.html>`__ (version 3.0.0 or later)
```

File: ccloud/docs/beginner-cloud.rst [hunk 2]
```diff
@@ -808,14 +791,13 @@ Additional Resources
 
 -  See additional Confluent CLI tutorials for producing and consuming events:
```
Comment (tikimims) [inline ccloud/docs/beginner-cloud.rst]:
```suggestion
-  For producing and consuming events, see the following |confluent-cli| tutorials: 
```

**Review by tikimims — APPROVED** [review]
LGTM @joel-hamill. Thank you :) Left a few suggestions.

**Review by davetroiano — APPROVED** [review]
LGTM

File: ccloud/docs/beginner-cloud.rst (hunk 1 above)
Comment (joel-hamill) [reply ccloud/docs/beginner-cloud.rst]: FYI i made this change over in the new location in docs-confluent-cli. https://github.com/confluentinc/docs-confluent-cli/pull/613

**Pull request merged**